fortuna/src/Login.js

```diff
@@ -1,49 +1,108 @@
 //@flow strict
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import ReactDOM from 'react-dom';
-import MarketPlace from './MarketPlace.js';
+import LoginPopup from './LoginPopup.js';
 import './Login.css';
-  
+const PORT = 3000;
```

Comment:
This works for now since this is the only API call, but this should probably get put in some constants file somewhere if it is going to be used all over

Reply:
Yup I agree only here to test how we are going to integrate the backend and frontend